src/main/resources/edu.umd.lib.wufooconnectorcamel.cfg

```diff
@@ -9,8 +9,7 @@ wufoo.servicename=wufoo-listener
 wufoo.handshake_key=testing
 
 # Wufoo Sysaid Field Mapping 
-#Form name : Ramesh Test - DSS
-wufoo.q1li2eu01gm1gnx.mapping=Wufoo-Sysaid-Mapping.properties
+wufoo.sysaid.mapping=q1li2eu01gm1gnx=/apps/git/wufoo-camel-connector/src/main/resources/Wufoo-Sysaid-Mapping.properties,q1li2eu01gm1gny=Wufoo-Sysaid-Mapping.properties
```

Comment:
Leave wufoo.sysaid.mapping blank and add a comment explaining how to add <form-hash>=<path/to/mapping/file> values to this property.